test/test_backends.py

```diff
@@ -443,19 +443,21 @@ def test_parse_hostspec(hostspec, expected):
 
 @pytest.mark.parametrize(
     'hostspec,pod,container,namespace,kubeconfig,context', [
-    ('kubectl://pod', 'pod', None, None, None, None),
-    ('kubectl://pod?namespace=n', 'pod', None, 'n', None, None),
-    ('kubectl://pod?container=c&namespace=n', 'pod', 'c', 'n', None, None),
-    ('kubectl://pod?namespace=n&kubeconfig=k', 'pod', None, 'n', 'k', None),
-    ('kubectl://pod?context=ctx&container=c', 'pod', 'c', None, None, 'ctx')
-])
-def test_kubectl_hostspec(hostspec, pod, container, namespace, kubeconfig, context):
-    backend = testinfra.get_host(hostspec).backend
-    assert backend.name == pod
-    assert backend.container == container
-    assert backend.namespace == namespace
-    assert backend.kubeconfig == kubeconfig
-    assert backend.context == context
+        ('kubectl://pod', 'pod', None, None, None, None),
+        ('kubectl://pod?namespace=n', 'pod', None, 'n', None, None),
+        ('kubectl://pod?container=c&namespace=n', 'pod', 'c', 'n', None, None),
+        ('kubectl://pod?namespace=n&kubeconfig=k', 'pod', None, 'n', 'k', None),
+        ('kubectl://pod?context=ctx&container=c', 'pod', 'c', None, None, 'ctx')
+    ]
+)
+def test_kubectl_hostspec(hostspec, pod, container, namespace, kubeconfig, 
+    context):
```

Comment:
Yes python format rules and related tools are a bit boring... Try this:

```suggestion
def test_kubectl_hostspec(
    hostspec, pod, container, namespace, kubeconfig, context,
):
```